domains/m1cro.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "m1cro-cat",
+        "email": "cybdevteam@hotmail.com"
+    },
+    "record": {
+        "A": ["76.76.21.61"]
```

Comment:
We do not support Vercel.

Reply:
ok, can I have github pages?

Reply:
> We do not support Vercel.

I deleted my site from vercel, then I will host on github pages then

Reply:
Do I have to do something, or do you yourself? I don't know much about github

Reply:
As soon as you get your GitHub page setup, you change it to a CNAME record and have it with your GitHub Domain. Then I'll just approve and you should be good.

Reply:
I hope I did everything right :)